tests/test_matprod.py

```diff
@@ -16,9 +24,10 @@ def simple_matprod(z):
 @pytest.mark.parametrize("antpairs", [True, False])
 @pytest.mark.parametrize("precision", [1, 2])
 @pytest.mark.parametrize(
-    "method", ["CPUMatMul", "CPUVectorDot", "GPUMatMul", "GPUVectorDot"]
+    "method",
+    ["CPUMatMul", "CPUVectorDot", "CPUMatChunk"],  # , "GPUMatMul", "GPUVectorDot"]
```

Comment:
you should leave in the GPU methods... they should be skipped automatically if you have no GPU

Reply:
done